--- a/__loading__.js
+++ b/__loading__.js
@@ -9,14 +9,7 @@ pc.script.createLoadingScreen(function (app) {
         var splash = document.createElement('div');
         splash.id = 'application-splash';
         wrapper.appendChild(splash);
-        splash.style.display = 'none';
-
-        var logo = document.createElement('img');
-        logo.src = ASSET_PREFIX + 'logo.png';
-        splash.appendChild(logo);
-        logo.onload = function () {
-            splash.style.display = 'block';
-        };
+        splash.style.display = 'block';

         var container = document.createElement('div');
         container.id = 'progress-bar-container';

--- a/__start__.js
+++ b/__start__.js
@@ -115,10 +115,14 @@
         return;
     }

+    if(window.BAHookAppCreated) window.BAHookAppCreated(app);
+
     app.configure(CONFIG_FILENAME, function (err) {
         if (err) {
             console.error(err);
         }
+
+        if(window.BAHookAppConfigured) window.BAHookAppConfigured(app);

         configureCss(app._fillMode, app._width, app._height);
         reflow();
@@ -136,6 +140,8 @@
                     console.error(err);
                 }

+                if(window.BAHookSceneLoaded) window.BAHookSceneLoaded(scene);
+
                 app.start();
             });
         });

