diff --git a/__loading__.js b/__loading__.js
index 610e064..fd1b171 100644
--- a/__loading__.js
+++ b/__loading__.js
@@ -9,14 +9,7 @@ pc.script.createLoadingScreen(function (app) {
         var splash = document.createElement('div');
         splash.id = 'application-splash';
         wrapper.appendChild(splash);
-        splash.style.display = 'none';
-
-        var logo = document.createElement('img');
-        logo.src = ASSET_PREFIX + 'logo.png';
-        splash.appendChild(logo);
-        logo.onload = function () {
-            splash.style.display = 'block';
-        };
+        splash.style.display = 'block';
 
         var container = document.createElement('div');
         container.id = 'progress-bar-container';
diff --git a/__start__.js b/__start__.js
index 70a4074..eec7a3f 100644
--- a/__start__.js
+++ b/__start__.js
@@ -1,4 +1,4 @@
-(function () {
+window._startpc = function (target, onAppCreated, onAppConfigured, onSceneLoaded) {
     var CANVAS_ID = 'application-canvas';
 
     var canvas, devices, app;
@@ -7,12 +7,11 @@
         canvas = document.createElement('canvas');
         canvas.setAttribute('id', CANVAS_ID);
         canvas.setAttribute('tabindex', 0);
-        // canvas.style.visibility = 'hidden';
 
         // Disable I-bar cursor on click+drag
         canvas.onselectstart = function () { return false; };
 
-        document.body.appendChild(canvas);
+        target.appendChild(canvas);
 
         return canvas;
     };
@@ -56,16 +55,6 @@
         var size = app.resizeCanvas(canvas.width, canvas.height);
         canvas.style.width = '';
         canvas.style.height = '';
-
-        var fillMode = app._fillMode;
-
-        if (fillMode == pc.FILLMODE_NONE || fillMode == pc.FILLMODE_KEEP_ASPECT) {
-            if ((fillMode == pc.FILLMODE_NONE && canvas.clientHeight < window.innerHeight) || (canvas.clientWidth / canvas.clientHeight >= window.innerWidth / window.innerHeight)) {
-                canvas.style.marginTop = Math.floor((window.innerHeight - canvas.clientHeight) / 2) + 'px';
-            } else {
-                canvas.style.marginTop = '';
-            }
-        }
     };
 
     var displayError = function (html) {
@@ -115,11 +104,15 @@
         return;
     }
 
+    onAppCreated(app);
+
     app.configure(CONFIG_FILENAME, function (err) {
         if (err) {
             console.error(err);
         }
 
+        onAppConfigured(app);
+
         configureCss(app._fillMode, app._width, app._height);
         reflow();
 
@@ -136,8 +129,10 @@
                     console.error(err);
                 }
 
+                onSceneLoaded(scene);
+
                 app.start();
             });
         });
     });
-}());
+};
